runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -713,6 +713,26 @@ public void scrollBy(int x, int y) {
         mContentView.scrollBy(x, y);
     }
 
+    public int computeHorizontalScrollRange() {
```

Comment:
Order:  computeHorizontalScrollOffset/computeHorizontalScrollRange, 
computeVerticalScrollExtent/computeVerticalScrollOffset/computeVerticalScrollRange.

And test case?